supervisor/homeassistant/websocket.py

```diff
@@ -279,3 +279,28 @@ def send_message(self, message: dict[str, Any]) -> None:
         if self.sys_core.state in CLOSING_STATES:
             return
         self.sys_create_task(self.async_send_message(message))
+
+    async def async_supervisor_event(
+        self, event: WSEvent, data: dict[str, Any] | None = None
+    ):
+        """Send a supervisor/event command to Home Assistant."""
+        try:
+            await self.async_send_message(
+                {
+                    ATTR_TYPE: WSType.SUPERVISOR_EVENT,
+                    ATTR_DATA: {
+                        ATTR_EVENT: event,
+                        ATTR_DATA: data or {},
+                    },
+                }
+            )
+        except HomeAssistantWSNotSupported:
+            pass
+        except HomeAssistantWSError as err:
+            _LOGGER.error(err)
```

Comment:
Let's add a message for humans and not only an err

Reply:
@pvizeli Changed both places (`async_supervisor_event` and `async_supervisor_update_event`) to this:
```py
_LOGGER.error("Could not send message to Home Assistant due to %s", err)
```
Sound good?

Reply:
There is no need for exception handling in both those, one of them should call the other

Reply:
Right that's true. I'll adjust it in a follow-up